src/main/kotlin/blackjack/GameController.kt

```diff
@@ -0,0 +1,22 @@
+package blackjack
+
+import blackjack.domain.betting.BettingResult
+import blackjack.domain.game.BlackjackGame
+import blackjack.domain.game.BlackjackParticipant
+import blackjack.domain.participant.Dealer
+import blackjack.domain.result.BlackjackResult
+import blackjack.view.InputView
+import blackjack.view.ResultView
+
+fun main() {
+    val names = InputView.getNames()
+    val participant = BlackjackParticipant.of(Dealer(), names, InputView::getBettingMoney)
```

Comment:
```suggestion
    val participant = BlackjackParticipant.of(
        dealer = Dealer(), 
        names = names, 
        input = InputView::getBettingMoney
    )
```
코틀린에는 argument name이라는 것이 있습니다.
argument name을 활용하면, 파라미터에 무엇을 넣어야할지 알 수 있어서 가독성이 좋아집니다.
파라미터가 3개 이상이거나, 함수명만 보고서 파라미터를 무엇을 넣어야할지 짐작하기 어려울 경우에는 argument name을 활용하는 것을 권장합니다. (이하 관련 내용 동일)